third-party/easyloggingpp/src/easylogging++.cc

```diff
@@ -2094,8 +2099,10 @@ Storage::Storage(const LogBuilderPtr& defaultLogBuilder) :
   addFlag(LoggingFlag::AllowVerboseIfModuleNotSpecified);
 #if ELPP_ASYNC_LOGGING
   installLogDispatchCallback<base::AsyncLogDispatchCallback>(std::string("AsyncLogDispatchCallback"));
+  std::cout << "Storage::ELPP_ASYNC_LOGGING" << std::endl;
```

Comment:
Remove couts or replace with profiling messages with conditional compilation for testing

Reply:
done